src/commands/set-permalink-structure.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Set Permalink Structure
+ *
+ * @param type - Permalink structure, should contain structure tags if not empty string
+ *
+ * @example
+ * ```
+ * cy.setPermalinkStructure('/%year%/%postname%/')
+ * ```
+ */
+export const setPermalinkStructure = (type = ''): void => {
+  cy.visit('/wp-admin/options-permalink.php');
+  cy.get('#permalink_structure').click().clear();
+  if ('' !== type) {
+    cy.get('#permalink_structure').click().type(type);
+  }
```

Comment:
@cadic Should we check for this earlier? If users pass the empty type, we do nothing.

Or if the type is required, let's remove the default value!

Setting the default to `/%postname%/` is another good direction IMO.

Reply:
Empty string is for "Plain" structure. I will remove the default value.

The "clear before type" workaround is required because Cypress doesn't allow to type empty string.